lib/rules/no-identical-title.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
```

Comment:
Please add a JSDoc fileoverview comment (see https://github.com/platinumazure/eslint-plugin-qunit/blob/master/lib/rules/literal-compare-order.js#L1-L4). You can even put your name down for author since you created this rule :smile: